rpi_backlight.py

```diff
@@ -38,97 +39,174 @@ def _get_value(name: str) -> str:
 
 def _set_value(name: str, value: Any) -> None:
     try:
-        with open(os.path.join(PATH, name), "w") as f:
+        with open(os.path.join(path, name), "w") as f:
             f.write(str(value))
     except (OSError, IOError) as err:
         if err.errno == 13:
             _perm_denied()
 
 
-def get_actual_brightness() -> int:
-    """Return the actual display brightness."""
-    return int(_get_value("actual_brightness"))
+def set_brightness_value(value: Any) -> None:
+    if mode == "MODE_TINKERBOARD":
```

Comment:
Please change it back to constants as I explained earlier.

Reply:
I was not able to get it to work with constants, only global variables. Maybe you could give it a shot. I spent an hour with constants before giving up

Reply:
Fixed